win32/sendmail.c

```diff
@@ -333,6 +333,33 @@ PHPAPI char *GetSMErrorText(int index)
 	}
 }
 
+/* strtok_r like, but recognizes quoted-strings */
+static char *find_address(char *list, char **state)
+{
+	zend_bool in_quotes = 0;
+	char *p = list;
+
+	if (list == NULL) {
+		if (*state == NULL) {
+			return NULL;
+		}
+		p = list = *state;
+	}
+	*state = NULL;
+	while ((p = strpbrk(p, ",\"\\")) != NULL) {
+		if (*p == '\\' && in_quotes) {
+			p++;
```

Comment:
I suspect this may end up reading beyond the end of the string for an `"\` at the end of the string, which will then skip over the terminating null byte.

Reply:
Indeed, it would. Thanks again!